github/admin.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import "fmt"
+
+// AdminService handles communication with the admin related methods of the
+// GitHub API. These API routes are normally only accessible for GitHub
+// Enterprise installations.
+//
+// GitHub API docs: https://developer.github.com/v3/enterprise/
```

Comment:
My intention was to add more routes from the enterprise/admin section at a later point, yep. There are other routes in the `admin/` namespace that I haven't added yet.